packages/ui-components/src/components/range/range.helper.ts

```diff
@@ -1,8 +1,9 @@
-export const getInputPercentageFromValue = (inputValue: string, min: number, max: number) => {
-	return (100 * (parseInt(inputValue) - min)) / (max - min);
+export const getInputPercentageFromValue = (inputValue: number, min: number, max: number) => {
+	return (100 * (inputValue - min)) / (max - min);
 };
 
 export const getOffset = (percentage: number) => {
-	const invertPercentage = 1 - percentage / 100;
-	return invertPercentage * 10 - 4;
+	const offSet = -2.5;
```

Comment:
can this be on a configuration with more meaning?

Reply:
Well.. for some reason the label above the button was not following the button properly with only the percentage, I suspect it's because the size of the button was changed, but I'm not sure. To be honest, I got this calculation by trial and error, it seems to work with every input I put.